scripts/cibuild.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+vagrant up --provision
```

Comment:
While [not explicitly stated here](https://github.com/azavea/architecture/blob/master/doc/arch/adr-0000-scripts-to-rule-them-all.md), generally the only script that wraps its commands in `vagrant ssh -c` is the `./scripts/setup`. All other scripts are intended to be run within the VM. It would be good to refactor these bash scripts to follow that convention. The climate API project is still a good example to base a refactor on.